14
subject 111312
no variables loaded
version 0
0
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/distributed/node.py:151: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 41907 instead
  warnings.warn(
parrallel submitted 11:13:01
parrallel finished 11:17:32
estimation_imag_fix_testretest.py:217: ComplexWarning: Casting complex values to real discards the imaginary part
  inv_JinvCeJ[i,:] = J[:,i].T * inv_DiagCe
estimation_imag_fix_testretest.py:230: ComplexWarning: Casting complex values to real discards the imaginary part
  P[:,i] = (J[:,i]*np.squeeze(inv_DiagCe))
estimation_imag_fix_testretest.py:250: ComplexWarning: Casting complex values to real discards the imaginary part
  g[i] = -0.5*np.exp(lembda[i])*np.trace(P[T*i:T*i+T][:, T*i:T*i+T])
estimation_imag_fix_testretest.py:255: ComplexWarning: Casting complex values to real discards the imaginary part
  g[i] = g[i] + g_rest
/users/k1201869/wang_model/code/model_fitting/wang_functions_imag_fix.py:502: ComplexWarning: Casting complex values to real discards the imaginary part
  tr[i] = np.sum(temp)
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/numpy/core/fromnumeric.py:87: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
  return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
[[-2.01343036]]
estimation_imag_fix_testretest.py:309: ComplexWarning: Casting complex values to real discards the imaginary part
  JinvCe[i,:] = J[:,i] * inv_DiagCe #;% J'%invCe <----- p x nT
/users/k1201869/wang_model/code/model_fitting/wang_functions_imag_fix.py:517: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
  sol = np.linalg.lstsq(A, b)[0]    # not under-determined
estimation_imag_fix_testretest.py:320: ComplexWarning: Casting complex values to real discards the imaginary part
  A_FDK[:,ppi] = np.squeeze(A) + np.squeeze(d_A) # %newton-gauss, fisher scoring, update Para_E
estimation_imag_fix_testretest.py:327: ComplexWarning: Casting complex values to real discards the imaginary part
  A_FDK[:,ppi] = np.squeeze(A) + d_A #newton-gauss, fisher scoring, update Para_E
distributed.client - ERROR - Failed to reconnect to scheduler after 10.00 seconds, closing client
_GatheringFuture exception was never retrieved
future: <_GatheringFuture finished exception=CancelledError()>
asyncio.exceptions.CancelledError
estimation_imag_fix_testretest.py:331: ComplexWarning: Casting complex values to real discards the imaginary part
  h_output_FDK[:,ppi], CC_check_FDK[ppi] = funcP(np.atleast_2d(Para_E_new).T)
estimation_imag_fix_testretest.py:334: ComplexWarning: Casting complex values to real discards the imaginary part
  dlddpara_FDK[:,:,ppi] = dlddpara
estimation_imag_fix_testretest.py:335: ComplexWarning: Casting complex values to real discards the imaginary part
  dldpara_FDK[:,ppi] = dldpara
[[-2.01343036]]
real won
goodness of fit correlation [0.06983997]
1
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/distributed/node.py:151: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 37395 instead
  warnings.warn(
parrallel submitted 11:18:23
parrallel finished 11:22:06
estimation_imag_fix_testretest.py:217: ComplexWarning: Casting complex values to real discards the imaginary part
  inv_JinvCeJ[i,:] = J[:,i].T * inv_DiagCe
estimation_imag_fix_testretest.py:230: ComplexWarning: Casting complex values to real discards the imaginary part
  P[:,i] = (J[:,i]*np.squeeze(inv_DiagCe))
estimation_imag_fix_testretest.py:250: ComplexWarning: Casting complex values to real discards the imaginary part
  g[i] = -0.5*np.exp(lembda[i])*np.trace(P[T*i:T*i+T][:, T*i:T*i+T])
estimation_imag_fix_testretest.py:255: ComplexWarning: Casting complex values to real discards the imaginary part
  g[i] = g[i] + g_rest
/users/k1201869/wang_model/code/model_fitting/wang_functions_imag_fix.py:502: ComplexWarning: Casting complex values to real discards the imaginary part
  tr[i] = np.sum(temp)
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/numpy/core/fromnumeric.py:87: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
  return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
[[-2.02348513]]
estimation_imag_fix_testretest.py:309: ComplexWarning: Casting complex values to real discards the imaginary part
  JinvCe[i,:] = J[:,i] * inv_DiagCe #;% J'%invCe <----- p x nT
/users/k1201869/wang_model/code/model_fitting/wang_functions_imag_fix.py:517: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
  sol = np.linalg.lstsq(A, b)[0]    # not under-determined
estimation_imag_fix_testretest.py:320: ComplexWarning: Casting complex values to real discards the imaginary part
  A_FDK[:,ppi] = np.squeeze(A) + np.squeeze(d_A) # %newton-gauss, fisher scoring, update Para_E
estimation_imag_fix_testretest.py:327: ComplexWarning: Casting complex values to real discards the imaginary part
  A_FDK[:,ppi] = np.squeeze(A) + d_A #newton-gauss, fisher scoring, update Para_E
distributed.client - ERROR - Failed to reconnect to scheduler after 10.00 seconds, closing client
_GatheringFuture exception was never retrieved
future: <_GatheringFuture finished exception=CancelledError()>
asyncio.exceptions.CancelledError
estimation_imag_fix_testretest.py:331: ComplexWarning: Casting complex values to real discards the imaginary part
  h_output_FDK[:,ppi], CC_check_FDK[ppi] = funcP(np.atleast_2d(Para_E_new).T)
estimation_imag_fix_testretest.py:334: ComplexWarning: Casting complex values to real discards the imaginary part
  dlddpara_FDK[:,:,ppi] = dlddpara
estimation_imag_fix_testretest.py:335: ComplexWarning: Casting complex values to real discards the imaginary part
  dldpara_FDK[:,ppi] = dldpara
[[-2.08593347]]
real won
goodness of fit correlation [0.1488504]
2
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/distributed/node.py:151: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 43822 instead
  warnings.warn(
parrallel submitted 11:22:57
